### PR TITLE
Update hipGetBackendNativeHandles use

### PR DESCRIPTION
**Pull request by pvelesko**

### CODE REVIEW

File: src/util.cpp [hunk 1]
```diff
@@ -45,13 +45,13 @@ hipblasCreate(hipblasHandle_t* handle)
...
     if(handle != nullptr)
     {
-        // HIP supports mutile backends hence query current backend name
-        auto backendName = hipGetBackendName();
         // Obtain the handles to the back handlers.
-        unsigned long handles[4];
-        int           nHandles = 4;
-        hipGetBackendNativeHandles((uintptr_t)NULL, handles, &nHandles);
-        *handle = H4I::MKLShim::Create(handles, nHandles, backendName);
+        int nHandles;
```
Comment (Sarbojit2019) [inline src/util.cpp:49]: These are backward compatibility breaking changes. We should add a check to verify chipStar is compatible. Think of a situation where chipStar runtime is old but hipBlas is new.

**Review by Sarbojit2019 — CHANGES_REQUESTED** [review]
Add a check to make sure chipStar is compatible.

**Comment by pvelesko** [comment]
made this backwards compatible with HIP 6.x

**Review by Sarbojit2019 — APPROVED** [review]
Looks ok to me

**Pull request merged**